From b8f9e1306d0f4350a5dea6884108603f8ffa8171 Mon Sep 17 00:00:00 2001
From: Rush Chen <rush.chen.wiwynn@gmail.com>
Date: Tue, 27 Aug 2024 16:17:47 +0800
Subject: [PATCH] configurations: yosemite4: Add PowerState to Medusa Board

To let psusensor get real value of MEDUSA_MBX_ sensors while
corresponding chassis power state is ON.

Test plan:
1. Modify property "CurrentPowerState" to
xyz.openbmc_project.State.Chassis.PowerState.Off, the sensor value of
corresponding slot should be nan.
2. Change property "CurrentPowerState" to
xyz.openbmc_project.State.Chassis.PowerState.Off, the sensor value of
corresponding slot should be real value.
---
 ...edusaboard_adc_rns_12vhsc_adi_48vhsc_inf.json | 16 ++++++++++++++++
 ...edusaboard_adc_rns_12vhsc_mps_48vhsc_adi.json | 16 ++++++++++++++++
 ...edusaboard_adc_rns_12vhsc_mps_48vhsc_inf.json | 16 ++++++++++++++++
 .../yosemite4_medusaboard_adc_rns_hsc_adi.json   | 16 ++++++++++++++++
 ...medusaboard_adc_ti_12vhsc_adi_48vhsc_inf.json | 16 ++++++++++++++++
 ...medusaboard_adc_ti_12vhsc_mps_48vhsc_adi.json | 16 ++++++++++++++++
 ...medusaboard_adc_ti_12vhsc_mps_48vhsc_inf.json | 16 ++++++++++++++++
 .../yosemite4_medusaboard_adc_ti_hsc_adi.json    | 16 ++++++++++++++++
 8 files changed, 128 insertions(+)

diff --git a/configurations/yosemite4_medusaboard_adc_rns_12vhsc_adi_48vhsc_inf.json b/configurations/yosemite4_medusaboard_adc_rns_12vhsc_adi_48vhsc_inf.json
index 10b3cf1..40bd043 100644
--- a/configurations/yosemite4_medusaboard_adc_rns_12vhsc_adi_48vhsc_inf.json
+++ b/configurations/yosemite4_medusaboard_adc_rns_12vhsc_adi_48vhsc_inf.json
@@ -146,6 +146,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "1",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB1_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
@@ -301,6 +303,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "2",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB2_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
@@ -456,6 +460,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "3",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB3_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
@@ -611,6 +617,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "4",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB4_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
@@ -766,6 +774,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "5",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB5_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
@@ -921,6 +931,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "6",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB6_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
@@ -1076,6 +1088,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "7",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB7_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
@@ -1231,6 +1245,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "8",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB8_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
diff --git a/configurations/yosemite4_medusaboard_adc_rns_12vhsc_mps_48vhsc_adi.json b/configurations/yosemite4_medusaboard_adc_rns_12vhsc_mps_48vhsc_adi.json
index 351cced..a882da7 100644
--- a/configurations/yosemite4_medusaboard_adc_rns_12vhsc_mps_48vhsc_adi.json
+++ b/configurations/yosemite4_medusaboard_adc_rns_12vhsc_mps_48vhsc_adi.json
@@ -663,6 +663,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "1",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB1_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
@@ -818,6 +820,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "2",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB2_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
@@ -973,6 +977,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "3",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB3_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
@@ -1128,6 +1134,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "4",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB4_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
@@ -1283,6 +1291,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "5",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB5_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
@@ -1438,6 +1448,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "6",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB6_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
@@ -1593,6 +1605,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "7",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB7_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
@@ -1748,6 +1762,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "8",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB8_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
diff --git a/configurations/yosemite4_medusaboard_adc_rns_12vhsc_mps_48vhsc_inf.json b/configurations/yosemite4_medusaboard_adc_rns_12vhsc_mps_48vhsc_inf.json
index e979d0b..9896fb0 100644
--- a/configurations/yosemite4_medusaboard_adc_rns_12vhsc_mps_48vhsc_inf.json
+++ b/configurations/yosemite4_medusaboard_adc_rns_12vhsc_mps_48vhsc_inf.json
@@ -353,6 +353,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "1",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB1_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
@@ -508,6 +510,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "2",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB2_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
@@ -663,6 +667,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "3",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB3_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
@@ -818,6 +824,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "4",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB4_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
@@ -973,6 +981,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "5",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB5_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
@@ -1128,6 +1138,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "6",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB6_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
@@ -1283,6 +1295,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "7",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB7_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
@@ -1438,6 +1452,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "8",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB8_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
diff --git a/configurations/yosemite4_medusaboard_adc_rns_hsc_adi.json b/configurations/yosemite4_medusaboard_adc_rns_hsc_adi.json
index a70f39f..66f3fe5 100644
--- a/configurations/yosemite4_medusaboard_adc_rns_hsc_adi.json
+++ b/configurations/yosemite4_medusaboard_adc_rns_hsc_adi.json
@@ -456,6 +456,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "1",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB1_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
@@ -611,6 +613,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "2",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB2_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
@@ -766,6 +770,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "3",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB3_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
@@ -921,6 +927,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "4",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB4_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
@@ -1076,6 +1084,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "5",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB5_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
@@ -1231,6 +1241,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "6",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB6_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
@@ -1386,6 +1398,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "7",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB7_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
@@ -1541,6 +1555,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "8",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB8_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
diff --git a/configurations/yosemite4_medusaboard_adc_ti_12vhsc_adi_48vhsc_inf.json b/configurations/yosemite4_medusaboard_adc_ti_12vhsc_adi_48vhsc_inf.json
index 3e2709d..5f41356 100644
--- a/configurations/yosemite4_medusaboard_adc_ti_12vhsc_adi_48vhsc_inf.json
+++ b/configurations/yosemite4_medusaboard_adc_ti_12vhsc_adi_48vhsc_inf.json
@@ -146,6 +146,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "1",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB1_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
@@ -301,6 +303,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "2",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB2_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
@@ -456,6 +460,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "3",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB3_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
@@ -611,6 +617,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "4",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB4_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
@@ -766,6 +774,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "5",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB5_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
@@ -921,6 +931,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "6",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB6_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
@@ -1076,6 +1088,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "7",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB7_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
@@ -1231,6 +1245,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "8",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB8_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
diff --git a/configurations/yosemite4_medusaboard_adc_ti_12vhsc_mps_48vhsc_adi.json b/configurations/yosemite4_medusaboard_adc_ti_12vhsc_mps_48vhsc_adi.json
index fdecea9..7a774da 100644
--- a/configurations/yosemite4_medusaboard_adc_ti_12vhsc_mps_48vhsc_adi.json
+++ b/configurations/yosemite4_medusaboard_adc_ti_12vhsc_mps_48vhsc_adi.json
@@ -673,6 +673,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "1",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB1_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
@@ -828,6 +830,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "2",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB2_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
@@ -983,6 +987,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "3",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB3_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
@@ -1138,6 +1144,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "4",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB4_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
@@ -1293,6 +1301,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "5",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB5_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
@@ -1448,6 +1458,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "6",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB6_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
@@ -1603,6 +1615,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "7",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB7_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
@@ -1758,6 +1772,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "8",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB8_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
diff --git a/configurations/yosemite4_medusaboard_adc_ti_12vhsc_mps_48vhsc_inf.json b/configurations/yosemite4_medusaboard_adc_ti_12vhsc_mps_48vhsc_inf.json
index a1c9c33..1e8bcf7 100644
--- a/configurations/yosemite4_medusaboard_adc_ti_12vhsc_mps_48vhsc_inf.json
+++ b/configurations/yosemite4_medusaboard_adc_ti_12vhsc_mps_48vhsc_inf.json
@@ -363,6 +363,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "1",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB1_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
@@ -518,6 +520,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "2",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB2_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
@@ -673,6 +677,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "3",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB3_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
@@ -828,6 +834,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "4",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB4_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
@@ -983,6 +991,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "5",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB5_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
@@ -1138,6 +1148,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "6",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB6_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
@@ -1293,6 +1305,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "7",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB7_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
@@ -1448,6 +1462,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "8",
+            "PowerState" : "ChassisOn",
             "Type": "MP5990",
             "iout1_Name": "MEDUSA_MB8_12VHSC_CURR_A_2",
             "iout1_Scale": 500,
diff --git a/configurations/yosemite4_medusaboard_adc_ti_hsc_adi.json b/configurations/yosemite4_medusaboard_adc_ti_hsc_adi.json
index b12a4c5..aa17479 100644
--- a/configurations/yosemite4_medusaboard_adc_ti_hsc_adi.json
+++ b/configurations/yosemite4_medusaboard_adc_ti_hsc_adi.json
@@ -456,6 +456,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "1",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB1_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
@@ -611,6 +613,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "2",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB2_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
@@ -766,6 +770,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "3",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB3_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
@@ -921,6 +927,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "4",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB4_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
@@ -1076,6 +1084,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "5",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB5_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
@@ -1231,6 +1241,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "6",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB6_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
@@ -1386,6 +1398,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "7",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB7_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
@@ -1541,6 +1555,8 @@
                     "Value": 10.091
                 }
             ],
+            "SlotId" : "8",
+            "PowerState" : "ChassisOn",
             "Type": "ADM1281",
             "iout1_Name": "MEDUSA_MB8_12VHSC_CURR_A_1",
             "iout1_Scale": 500,
-- 
2.25.1

